DMX.Core/DMX.Core.API/Program.cs

```diff
@@ -0,0 +1,25 @@
+// --------------------------------------------------------------
+// Copyright (c) Michael Mendelsohn All rights reserved.
+// Liscensed under the MIT License.
+// See License.txt in the project root for licesnse infromation.
+// --------------------------------------------------------------
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace DMX.Core.API
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
```

Comment:
Fat arrow a single line method - for instance:

instead of:
```csharp
public static string GetName()
{
    return "Geetika";
}
```

have it say:
```csharp
public static string GetName() => "Geetika";
```
If the return type is too long (longer than 120 characters) then you may do this:
```csharp
public static string GetName() =>
    AVeryLongNamedMethodCall();
```
Mind the indentations.